README.md

```diff
@@ -0,0 +1,43 @@
+# HLTRho: Spectral densities from two-point correlation functions
+
+HLTRho is a Python library for operating spectral density reconstruction and
+related studies, by using the HLT (Hansen-Lupo-Tantalo) method.
+
+## Authors
+
+Niccolo' Forzano, Alessandro Lupo.
+
+## Installation
+
+Being within the root directory rhos/, where 'pyproject.toml' is located, one can build the package
+
+```bash
+python3 -m build
```

Comment:
Implemented. I have changed the name in hltrho. I did not think carefully at the name, and I did not consult Alessandro for this one, he might have opinions and proposal. I have no strong opinions for this one.

Reply:
how about "lattice-inverse-problem" 💋💋💋
are hyphens allowed? i think hlt is a bit restrictive because more solutions are implemented (and I feel dorky having my name in it)

Reply:
I agree with you saying that hltrho is too restrictive. I am fine with your choice.

Reply:
You can have hyphens in a PyPI name but not in Python (as `-` is a minus sign); the convention would then be to use underscores instead in the Python name.

Is `lattice_inverse_problem` potentially _too_general? Could something like `densimeter` work? (Sufficiently uncommon word as to not have too much trouble finding things on search engines, and gives the indication of (spectral) densities…)

Reply:
I Have used 'LatticeInverseProblem'. Does this look too bad done this way?

Reply:
Package names in Python conventionally use `snake_case`; `PascalCase` is normally used for class names instead. This isn't a hard an fast rule, so this wouldn't necessarily be the only package with that convention, but I'm usually a fan of trying to match the most commonly-used conventions where possible.

Reply:
@LupoA , would you rather '''densimeter''' or '''lattice_inverse_problem'''? I am happy with either! Ed here probably has a point in his message.

Reply:
I think lattice_inverse_problem is not too general since this seems to be the name every one uses in the community. If we want to avoid underscores I propose the name "**lsdensities**" for lattice spectral densities. Do you like it?

Reply:
[this is hard](https://www.youtube.com/watch?v=eDNKfL0o32Q)

Reply:
The three hardest problems in programming are naming things and off-by-one errors :)

While underscores are fine, `lsdensities` is quicker to type and more uniquely searchable, so that'd be my preference.

Reply:
`lsdensities` it is! go ahead @nickforce98 and sorry for changing my mind

Reply:
This is hard indeed :laughing: 

Anyways, implemented.

Reply:
I suppose I should change the name of the repository accordingly, but that can happen after this PR. I don't suppose it would break things